meson.build

```diff
@@ -0,0 +1,54 @@
+project(
+  'edlib',
+  'cpp', 'c',
+  version : '1.2.6',
+  default_options : [
+    'buildtype=release',
+    'warning_level=3',
+    'cpp_std=c++17',
+    'b_ndebug=if-release'],
+  license : 'MIT',
+  meson_version : '>= 0.52.0',
+)
+edlib_lib = library('edlib',
+  sources : ['edlib/src/edlib.cpp'],
```

Comment:
`sources : files(['edlib/src/edlib.cpp']),`

Reply:
But it works without `files()` already, right? When would this matter?

Reply:
1. if you ever rename a file or are missing a new one, `files()` will catch it. You want failures at `meson configure` time, not when calling `ninja`. Fail-fast
2. it can be used across directories
3. it's the standard idiom recommended by meson upstream